tests/accessor/local_accessor_access_among_work_items.h

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests that the sycl::local_accessor can access to the memory shared
+//  among work-items.
+//
+*******************************************************************************/
+#ifndef SYCL_CTS_LOCAL_ACCESSOR_ACCESS_AMONG_WORK_ITEMS_H
+#define SYCL_CTS_LOCAL_ACCESSOR_ACCESS_AMONG_WORK_ITEMS_H
+#include "accessor_common.h"
+
+namespace local_accessor_access_among_work_items {
+using namespace sycl_cts;
+using namespace accessor_tests_common;
+
+/**
+ * @brief Provides a functor that provides verification that local_accessor can
+ *        access to the memory shared among work-items
+ * @tparam T Current data type
+ * @tparam DimensionTypeT Current current dimension size
+ */
+template <typename T, typename DimensionTypeT>
+class run_test {
+  static constexpr int Dimension = DimensionT::value;
+
+ public:
+  /**
+   * @brief Functor that provides verification that local_accessor can access to
+   *        the memory shared among work-items
+   * @param type_name Current data type string representation
+   */
+  void operator()(const std::string& type_name) {
+    auto queue = util::get_cts_object::queue();
+
+    auto section_name = get_section_name<Dimension>(
+        type_name,
+        "Verify possibility to access to the memory shared among work-items. "
+        "[local_accessor]");
```

Comment:
```suggestion
        "Verify possibility to access the memory shared among work-items. "
        "[local_accessor]");
```

Reply:
This comment has been applied [here](https://github.com/KhronosGroup/SYCL-CTS/pull/364/commits/bb0394a3416ce372412bd19c44da15e021bf8702#diff-cbe2db4042ddaa9f8e561ec7b67998e37f2ade0860b556328eacd9374694816cL38-R38)